ddtrace/settings/profiling.py

```diff
@@ -315,6 +362,12 @@ class ProfilingConfigExport(En):
 
 config = ProfilingConfig()
 
+# If during processing we discover that the configuration was injected, we need to do a few things
+# - Mark it as such
+# - Force libdd to be enabled, disabling the profiler otherwise the service might crash
+#   (this is done in the _is_libdd_required function)
+config.injected = _check_for_injected()
```

Comment:
```suggestion
config._injected = _check_for_injected()
```
no need to leak to the public API?

Reply:
Good idea, fixed this and the references